jekyll/_cci2/aws.md

```diff
@@ -105,7 +105,9 @@ Have available the following information and policies before starting the Previe
 ## Installation with Terraform
 1. Clone the [Setup](https://github.com/circleci/enterprise-setup) repository (if you already have it cloned, make sure it is up-to-date and you are on the `master` branch: `git checkout master && get pull`).
 2. Run `make init` to init `terraform.tfvars` file (your previous `terraform.tfvars` if any, will be backed up in the same directory).
-3. Fill `terraform.tfvars` with appropriate values.
+3. Fill `terraform.tfvars` with appropriate AWS values for section 1. 
+4. Specify a `circle_secret_passphrase` in section 2, replacing `...` with alpha numeric characters. Passprhase cannot be empty.
+5. Specify the instance size for your Nomad Clients. By default, the value specified in the `terraform.tfvars` file for Nomad Clients that get deployed are `m4.xlarge` instances which support three clients per server.  To increase the number of clients per server to support mutiple parallel builds, for example, modify section 2 of the `enterprise-setup/terraform.tfvars` file to specify a larger instance type for the `nomad_client_instance_type`. Using an `m4.2xlarge` instance with 8 CPUs and 32GB RAM is best practice for Nomad Clients. **Note:** The `builder_instance_type` is only used for 1.0 and is disabled by default in section 3. 
```

Comment:
'clients per server' this is not the right language.... each **instance** is a nomad client. we are talking about number of CircelCI jobs (ie, nomad jobs, under the hood) that each **client** is capable of handling